packages/anoncreds/src/services/registry/CredentialDefinitionOptions.ts

```diff
@@ -20,18 +20,18 @@ export interface RegisterCredentialDefinitionOptions {
 }
 
 export interface RegisterCredentialDefinitionReturnStateFailed extends AnonCredsOperationStateFailed {
-  credentialDefinition: AnonCredsCredentialDefinition
-  credentialDefinitionId?: string
+  credentialDefinition?: AnonCredsCredentialDefinition
+  credentialDefinitionId: string | null
```

Comment:
Same comment here about the `| null`.

Reply:
In this case we could do the `?`. Will change.